Automattic/wp-calypso
# Enable new migration instructions screen for everyone

Related to #91815
Based off of #92766

## Proposed Changes

* Remove English check and open feature for any user.
* Remove old files.

## Why are these changes being made?


* We are waiting for translations to remove this check. When they are done, this PR will be deployed.

## Testing Instructions



* Change your language to something different than English (`https://wordpress.com/me/account `).
* Test the 3 flows, and make sure the new site instructions is properly displayed:
  * `http://calypso.localhost:3000/start`
  * `http://calypso.localhost:3000/setup/migration-signup`
  * `http://calypso.localhost:3000/setup/hosted-site-migration?from={FROM_URL}`

## Pre-merge Checklist
